setup-env.template

```diff
@@ -8,17 +8,24 @@
 
 export DEPLOYMENT_NAME=cluster-name  # roman, jwebbinar, tike
 export ENVIRONMENT=sandbox           # sandbox, dev, test, prod, int
-export CAL_VERSION=latest            # e.g. roman or jwst x.y.z CAL release, latest, or none
+export CAL_VERSION=none              # e.g. roman or jwst x.y.z CAL release, latest, or none
```

Comment:
why none in here?

Reply:
"none" is the CAL_VERSION mode in which image-update does not try to update *floating* requirements for Roman.   For roman,  floating requirements are "official" but vary between highly pinned and development pinning.  Since the dev requirements required small mods,  I turned off the auto-update which would replace them on the next build.   Note that it has no effect on frozen or chilly requirements which can still be computed and saved.